{"name": "用例6：观众端查看主播端发言", "status": "broken", "statusDetails": {"message": "FileNotFoundError: [Errno 2] No such file or directory: 'C:\\\\Users\\\\pgk bqt\\\\PycharmProjects\\\\Chamet_project\\\\common/screenshots/party_ten/party_ten_2023-06-26 11_53_36.png'", "trace": "self = <test_case.test_10party_mult.Test_multchamet object at 0x0000018BB81C7D10>\n\n    @allure.severity(allure.severity_level.BLOCKER)\n    @allure.title('用例6：观众端查看主播端发言')\n    def test_TenParty_006(self):\n        logging.info('===用例6：观众端查看主播端发言===')\n        try:\n            audience_chat_area = self.multaudience.audience_chat_areatext(-1)\n>           assert audience_chat_area == \"hello\"\nE           AssertionError: assert '[level]  qqqq5edc29 加入了房间' == 'hello'\nE             - hello\nE             + [level]  qqqq5edc29 加入了房间\n\ntest_10party_mult.py:132: AssertionError\n\nDuring handling of the above exception, another exception occurred:\n\nself = <test_case.test_10party_mult.Test_multchamet object at 0x0000018BB81C7D10>\n\n    @allure.severity(allure.severity_level.BLOCKER)\n    @allure.title('用例6：观众端查看主播端发言')\n    def test_TenParty_006(self):\n        logging.info('===用例6：观众端查看主播端发言===')\n        try:\n            audience_chat_area = self.multaudience.audience_chat_areatext(-1)\n            assert audience_chat_area == \"hello\"\n            logging.info('===断言成功，观众端可看到主播端发言===')\n        except AssertionError as e:\n            logging.info('===断言失败===')\n>           screen_name = self.multaudience.screenshot('party_ten')\n\ntest_10party_mult.py:136: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <common.partytencommon.multparty_ten_audience.Multaudience object at 0x0000018BB81EF090>, module = 'party_ten'\n\n    def screenshot(self, module):\n        time = self.getTime()\n        image_file = os.path.dirname(os.path.dirname(__file__)) + '/screenshots/party_ten/%s_%s.png' % (module, time)\n        self.driver.get_screenshot_as_file(image_file)\n>       with open(image_file, mode='rb') as f:\nE       FileNotFoundError: [Errno 2] No such file or directory: 'C:\\\\Users\\\\pgk bqt\\\\PycharmProjects\\\\Chamet_project\\\\common/screenshots/party_ten/party_ten_2023-06-26 11_53_36.png'\n\n..\\common\\partytencommon\\multparty_ten_audience.py:76: FileNotFoundError"}, "attachments": [{"name": "log", "source": "916de685-57cc-4091-9f57-1f319f9c5634-attachment.txt", "type": "text/plain"}], "start": 1687751615403, "stop": 1687751616594, "uuid": "eeb5a3c4-f89c-4fc1-b18a-ecbd8961c44b", "historyId": "22ebee5ff667a4a6c5ce984a9397455d", "testCaseId": "22ebee5ff667a4a6c5ce984a9397455d", "fullName": "test_10party_mult.Test_multchamet#test_TenParty_006", "labels": [{"name": "severity", "value": "blocker"}, {"name": "suite", "value": "test_10party_mult"}, {"name": "subSuite", "value": "Test_multchamet"}, {"name": "host", "value": "DESKTOP-8OL56K6"}, {"name": "thread", "value": "9316-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_10party_mult"}]}